source/quick-start.txt

```diff
@@ -50,6 +50,12 @@ If you are using `Gradle <https://gradle.org/>`__, add the following to your
 
 .. include:: /includes/quick-start/gradle-versioned.rst
 
+.. tip:: Download the JAR File
+   
+   You can directly download the JAR file for the {+driver-short+}
```

Comment:
Suggestion:
I think "directly" could provide contrast when compared with another option, but I think omitting it when there isn't a comparison being made would be better.

"You can download..."